src/views/mainLayout.js

```diff
@@ -12,18 +12,18 @@ const mainLayout = data => `
 <!doctype html>
 <html lang=${getLocale()}>
   <head>
-    <title>${getMessage('brand-fx-monitor')}</title>
+    <title>${data.meta?.title ?? getMessage('brand-fx-monitor')}</title>
     <style>html {display: none;}</style>
 
     <meta charset='utf-8'>
     <meta name='viewport' content='width=320, initial-scale=1'>
-    <meta name='description' content='${getMessage('meta-desc')}'>
+    <meta name='description' content='${data.meta?.socialDescription ?? getMessage('meta-desc-2')}'>
     <meta name='twitter:card' content='summary_large_image'>
-    <meta name='twitter:title' content='${getMessage('brand-fx-monitor')}'>
-    <meta name='twitter:description' content='${getMessage('meta-desc')}'>
+    <meta name='twitter:title' content='${data.meta?.socialTitle ?? getMessage('brand-fx-monitor')}'>
+    <meta name='twitter:description' content='${data.meta?.socialDescription ?? getMessage('meta-desc-2')}'>
     <meta name='twitter:image' content='${AppConstants.SERVER_URL}/images/og-image.webp'>
-    <meta property='og:title' content='${getMessage('brand-fx-monitor')}'>
-    <meta property='og:description' content='${getMessage('meta-desc')}'>
+    <meta property='og:title' content='${data.meta?.socialTitle ?? getMessage('brand-fx-monitor')}'>
+    <meta property='og:description' content='${data.meta?.socialDescription ?? getMessage('meta-desc-2')}'>
```

Comment:
per @pdehaan 's nit above, mainLayout might also benefit from variable consolidation above the template?

Reply:
Done — see: https://github.com/mozilla/blurts-server/pull/3009#discussion_r1177714203.